web/frontend/angular/src/app/dashboard-pages/notifications/notifications.component.ts

```diff
@@ -1,12 +1,60 @@
 import {Component, OnInit} from "@angular/core"
+import {ApisService} from "src/app/apis/apis.service"
 
 @Component({
     selector: "app-notifications",
     templateUrl: "./notifications.component.html",
     styleUrls: ["./notifications.component.scss"],
 })
 export class NotificationsComponent implements OnInit {
-    constructor() {}
+    constructor(private apiservice: ApisService) {}
 
-    ngOnInit(): void {}
+    ngOnInit(): void {
+        this.fetchTest()
+    }
+
+    addNotification(data: any) {
+        this.apiservice.createNotification(data).subscribe(
+            (res: any) => {
+                console.log(res)
+                if (res.status == 200) {
+                    let notification = document.getElementById("notification")
+                    notification?.classList.remove("d-none")
```

Comment:
This endpoint returns 201 on success. Also, there needs to be user feedback for success e.g. displaying some message saying the notification was created.